src/style.css

```diff
@@ -0,0 +1,130 @@
+html {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --btn-color: #00acdc;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
add box-sizing to all elements on the page (* selector) and remove everywhere else